developer_guides/add_new_arch.rst

```diff
@@ -0,0 +1,34 @@
+.. _add_new_arch:
+
+Adding a new DSP architecture to SOF
+====================================
+
+This is not yet a guide for architecure porting, but in general are two ways to 
+add support for new DSP architectures to SOF.
```

Comment:
This is not yet a guide for architecure porting, but in general, you can add support
for new DSP architectures to SOF in the following two ways: